services/src/datasets/external/gbif.rs

```diff
@@ -384,6 +587,106 @@ impl LayerCollectionProvider for GbifDataProvider {
             metadata: Default::default(),
         })
     }
+
+    async fn get_search_capabilities(&self) -> Result<SearchCapabilities> {
+        Ok(SearchCapabilities {
+            caps: vec![
+                SearchCapability {
+                    search_type: SearchType::FULLTEXT,
+                    autocomplete: true, // TODO have to see whether its too slow
+                    filters: None,
+                },
+                SearchCapability {
+                    search_type: SearchType::PREFIX,
+                    autocomplete: true, // TODO this should work fine with index
+                    filters: None, // TODO this is not needed in the current design since any filters applied are already part of the GBIF provider collection ID, so should we implement it now at all?
```

Comment:
idk, could be special facets for something

Reply:
I can tell you now that users would want species name (and maybe genus or family).

Otherwise, you would need to browse into some cat family tree or meat eating animals to only look for species names.